### PR TITLE
Feat/random bugfixes

### PR DESCRIPTION
**Pull request by nicksantan**

### CODE REVIEW

File: Assets/Scripts/ChallengeManagerScript.cs [hunk 1]
```diff
@@ -166,6 +166,11 @@ public void UpdateChallengeText(string newText)
         challengeTitle.GetComponent<Text>().text = newText;
         // TODO: Make a helper function to format the challenge number string
         challengeNumber.GetComponent<Text>().text = "CHALLENGE 0" + (DataManagerScript.challengeType + 1);
+        // Special case for challenge 10
+        if (DataManagerScript.challengeType == 9)
+        {
+            challengeNumber.GetComponent<Text>().text = "CHALLENGE 10";
+        }
```
Comment (runningjump) [inline Assets/Scripts/ChallengeManagerScript.cs]: If you wanted to make this more extensible, I think you could pad a zero like: 
```
int value = DataManagerScript.challengeType;
int paddedLength= 2;
int decimalLength = paddedLength - value.ToString("D").Length;
string paddedString = value.ToString("D" + decimalLength.ToString());
```
I adapted this [from an example](https://docs.microsoft.com/en-us/dotnet/standard/base-types/how-to-pad-a-number-with-leading-zeros) so I'm not sure if it works as is, but something like that.

File: Assets/Scripts/PlayerController.cs [hunk 2]
```diff
@@ -558,7 +558,7 @@ public void WinRumble()
         // This joystick - 1 shit has got to go.
         if (JoystickLayerManager.Instance != null && !isAI)
         {
-            JoystickLayerManager.Instance.ActivateLargeRumble(joystick - 1, 3f);
+            JoystickLayerManager.Instance.ActivateLargeRumble(joystick - 1, 2f);
```
Comment (runningjump) [inline Assets/Scripts/PlayerController.cs:561]: I didn't see this method, so neat!

**Review by runningjump — APPROVED** [review]
Looks good to me dude, my only note is making the challenge number stringification more universal. Take as thy will

File: Assets/Scripts/PlayerController.cs (hunk 2 above)
Comment (nicksantan) [reply Assets/Scripts/PlayerController.cs:561]: Yeah, check out the JoystickLayerManager and FilesystemLayer when you get a chance!

File: Assets/Scripts/ChallengeManagerScript.cs (hunk 1 above)
Comment (nicksantan) [reply Assets/Scripts/ChallengeManagerScript.cs]: Cool, made this change -

**Pull request merged**